src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -62,12 +63,28 @@ static int Last2DigitsTo0(int versionBuild)
                     .Select(p => Path.Combine(p, _sdkVersionBand))
                     .Append(manifestDirectory).ToArray();
             }
+            else if (userManifestsDirectory != null && Directory.Exists(userManifestsDirectory))
+            {
+                _manifestDirectories = new[] { manifestDirectory, userManifestsDirectory };
+            }
```

Comment:
I think that if the `DOTNETSDK_WORKLOAD_MANIFEST_ROOTS` environment variable is set, we should still look in the user local path.